release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2716,6 +2716,73 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[id="ocp-4-15-11-bug-fixes"]
+==== Bug fixes
+
+* Previously, for {product-title} deployments on {rh-openstack-first}, the `MachineSet` object did not correctly apply the value for the `Port Security` parameter. This mean the `Port_Security` parameter was wrong. With this release, the `MachineSet` object applies the `port_security_enabled` flag as expected (link:https://issues.redhat.com/browse/OCPBUGS-30857[*OCPBUGS-30857*])
+
+* Previously, the image registry did not support AWS region `ca-west-1`. With this release, the image registry can now be deployed in this region. (link:https://issues.redhat.com/browse/OCPBUGS-31641[*OCPBUGS-31641*])
+
+* Previously, a Redfish virtual media Hewlett Packard Enterprise (HPE) integrated Lights Out (iLO) 5 bare-metal machine's compression was forcibly disabled to workaround other unrelated issues in different hardware models. This caused `FirmwareSchema` resource to be missing from each iLO 5 bare-metal machine. Each machine needs compression to fetch message registries from their Redfish Baseboard Management Controller (BMC) endpoints. With this release, each iLO 5 bare-metal machine that needs the `FirmwareSchema` resource does not have compression forcibly disabled. (link:https://issues.redhat.com/browse/OCPBUGS-31686[*OCPBUGS-31686*])
+
+* Previously, nodes of paused `MachineConfigPools` might be incorrectly have their pause status dropped when performing a cluster update. With this release, nodes of paused `MachineConfigPools` correctly stay paused when performing a cluster update. (link:https://issues.redhat.com/browse/OCPBUGS-31839[*OCPBUGS-31839*])
+
+* Previously, newer versions of Redfish used Manager resources to deprecate the Uniform Resource Identifier (URI) for the RedFish Virtual Media API. This caused any hardware that used the newer Redfish URI for Virtual Media to not be provisioned. With this release, the Ironic API identifies the correct Redfish URI to deploy for the RedFish Virtual Media API so that hardware relying on either the deprecated or the newer URI could be provisioned.  (link:https://issues.redhat.com/browse/OCPBUGS-31830[*OCPBUGS-31830*])
+
+* Previously, the Cloud Credential Operator (CCO) checked for a non-existent `s3:HeadBucket` permission during the validation checks in mint mode that resulted in a failed cluster installation. With this release, CCO removes the validation check for this non-existing permission so that validation checks pass in mint mode and the cluster installation does not fail. (link:https://issues.redhat.com/browse/OCPBUGS-31924[*OCPBUGS-31924*])
+
+* Previously, a new Operator Lifecycle Manager (OLM) Operator that upgraded to {product-title} {product-version} resulted in failure because important resources were not injected into the upgrade operation. With this release, these resources are now cached so that newer OLM Operator upgrade can succeed. (link:https://issues.redhat.com/browse/OCPBUGS-32311[*OCPBUGS-32311*])
```

Comment:
```suggestion
* Previously, a new Operator Lifecycle Manager (OLM) Operator that upgraded to {product-title} {product-version} resulted in failure because important resources were not injected into the upgrade operation. With this release, these resources are now cached so that newer OLM Operator upgrades can succeed. (link:https://issues.redhat.com/browse/OCPBUGS-32311[*OCPBUGS-32311*])
```
Should it be {product-version} or should it be 4.15.3 like it is in the bug?

Reply:
Good point. Updated.